Dockerfile

```diff
@@ -19,8 +19,8 @@
 
 # Basic deps
 ARG CONTAINERD_VERSION=v1.7.16
-ARG RUNC_VERSION=v1.1.12
-ARG CNI_PLUGINS_VERSION=v1.4.1
+ARG RUNC_VERSION=v1.1.14
+ARG CNI_PLUGINS_VERSION=v1.5.1
 
 # Extra deps: Build
 ARG BUILDKIT_VERSION=v0.12.5
```

Comment:
Probably we should update `CONTAINERD_VERSION`, `BUILDKIT_VERSION`, `ROOTLESSKIT_VERSION`, etc.

Reply:
sure i can update those.
Probably also the test.yaml with the new versions.

Reply:
For the ROOTLESSKIT does the version update be limited to minor version or we also update major?

Reply:
nvm its a minor version upgrade will include it.